src/renderer/themes.css

```diff
@@ -261,6 +271,45 @@ it can be safely elided. This looks quite pleasant on this theme. */
   --logo-text: url("../../_icons/textNordicLightSmall.png");
 }
 
+.gruvboxDark {
+  --primary-text-color: #ebdbb2;
+  --secondary-text-color: #fbf1c7;
+  --tertiary-text-color: #ebdbb2;
+  --title-color: #ebdbb2;
+  --bg-color: #1d2021;
+  --favorite-icon-color: #fabd2f;
+  --card-bg-color: #282828;
+  --secondary-card-bg-color: #282828;
+  --scrollbar-color: #32302f;
```

Comment:
**issue (blocking):** Please choose a color that has [an acceptable color contrast](https://webaim.org/resources/contrastchecker/) (>=4.5:1) with the background (`--bg-color`). This scrollbar can be hard to find even for non-colorblind users.